app/src/main/res/layout-land/topup_pending_user_payment_view.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
The layout should be:
- Title and animation on top
- Text points in the middle
- Button on the bottom
All with the standard margins applied
The same logic should be applied in the landscape layout